tests/sessions_tests/tests.py

```diff
@@ -479,8 +494,12 @@ def count_sessions():
         other_session.set_expiry(-3600)
         other_session.save()
 
+        other_session2 = self.backend()
```

Comment:
add comment to match others? `# One object in the present without an expiry (should be deleted since its modification time + SESSION_COOKIE_AGE will be in the past when clearsessions runs) `